pre-commit

```diff
@@ -10,10 +10,14 @@ if [ -z "${STAGED_FILES}" ]; then
   exit 0;
 fi
 
-poetry run poe type_check "${STAGED_FILES}"
+poetry run poe type_check $(echo "${STAGED_FILES}")
```

Comment:
This didn't work properly as well, I didn't notice the error message in the cluttered output. It was saying something like this:
```
File 'path1
path2
path3' not found
```
So it cannot handle the `\n`-separated file list, it has to be space-split (which is done by using `echo`)